### ISSUE TITLE
Allow a window to be "locked" - i.e. TW is disabled for that window

### ISSUE THREAD

**Issue by attiks** [tabwrangler/tabwrangler#13]
It looks like the tabs are remembered by index, I have to windows each with tabs, but the Tab Lock is apparently the same for both windows?

**Comment by eorroe** [comment]
+1 on pause per window

**Comment by marijuanatech** [comment]
I know it's semantics, but I'd prefer "Locking Windows" rather than "Pausing Windows" for consistency's sake.

Is anyone working on this? I was going to mess around with it this coming week.

**Comment by ssorallen** [comment]
@marijuanatech No one has look at this feature. Happy to take any contributions.

**Comment by KTamas** [comment]
@marijuanatech yeah, feel free to submit a PR for this, it seems like something that'd be valuable for many people.
